api/src/interpreter/entities/interpreter.entity.ts

```diff
@@ -109,9 +109,9 @@ export class InterpreterEntity {
       id: this.id,
       firstName: this.firstName,
       lastName: this.lastName,
-      languages: this.languages.map((intLang: InterpreterLanguageEntity) =>
-        intLang.toResponseObject(),
-      ),
+      languages: this.languages
+        .map((intLang: InterpreterLanguageEntity) => intLang.toResponseObject())
+        .sort((a, b) => a.level - b.level),
```

Comment:
sorry, I think there's no feature from the queryBuilder yet.

https://github.com/typeorm/typeorm/issues/2620